probml_utils/dp_mixgauss_utils.py

```diff
@@ -0,0 +1,110 @@
+import jax.numpy as jnp
+from jax import random
+from collections import namedtuple
+from multivariate_t_utils import log_predic_t
+
+
+def dp_mixture_simu(N, alpha, H, key):
+    """
+    Generating samples from the Gaussian Dirichlet process mixture model.
+    We set the base measure of the DP to be Normal Inverse Wishart (NIW)
+    and the likelihood be multivariate normal distribution 
+    ------------------------------------------------------
+    N: int 
+        Number of samples to be generated from the mixture model
+    alpha: float
+        Concentration parameter of the Dirichlet process 
+    H: object of NormalInverseWishart
+        Base measure of the Dirichlet process
+    key: jax.random.PRNGKey
+        Seed of initial random cluster
+    --------------------------------------------
+    * array(N):
```

Comment:
Specify that these are return parameters, and give the variable names (eg `Z: array(N): ...`)